### ISSUE TITLE
Add option for HTML emails - or make all emails HTML by default

### ISSUE THREAD

**Issue by q2apro** [q2a/question2answer#97]
https://github.com/q2a/question2answer/blob/master/qa-include/qa-app-emails.php#L106

should be changed from 

```
'html' => false,
```

to `true`. 

Or alternatively there should be an option in the admin panel, maybe at admin/emails

This bothers me in every new q2a installation ... 

Thanks, Kai

**Comment by q2apro** [comment]
Again and again I run into this issue, please finally add this option. This is a qa_opt one liner here: 
https://github.com/q2a/question2answer/blob/dev/qa-include/app/mailing.php#L130

And a setting under admin... 

Feels very frustrating to hack the core again and again...

**Comment by arjunsuresh** [comment]
Agree. HTML email must be default these days. Trying to build one to support all email templates for Q2A.
